README.md

```diff
@@ -6,24 +6,34 @@
 <img alt="Build Status" src="https://prow.k8s.io/badge.svg?jobs=ci-kind-build">
 </a> <a href="https://goreportcard.com/report/sigs.k8s.io/kind"><img alt="Go Report Card" src="https://goreportcard.com/badge/sigs.k8s.io/kind" /></a>
 
+A tool for running local Kubernetes clusters using Docker container "nodes".  
+Pre-Request Installations: [Go] and [Docker] <br>
+  ```go
+ $ go get sigs.k8s.io/kind
+ $ kind create cluster
+ $ kind delete cluster
+ ```
+For more usage see [the documentation][user guide] or run `kind [command] --help`
+ 
+<img src="https://gist.githubusercontent.com/BenTheElder/621bc321fc6d9506fd936feb36d32dd0/raw/7fe14e9d0929cab428929ca6c501abc990c07359/kind-create-cluster.gif" alt="2x speed `kind create cluster` demo" /> <br>
 
-`kind` is a tool for running local Kubernetes clusters using Docker container "nodes".  
-`kind` is primarily designed for testing Kubernetes 1.11+, initially targeting the [conformance tests].
+ 
 
-If you have [go] and [docker] installed `go get sigs.k8s.io/kind && kind create cluster` is all you need!
```

Comment:
`&&` works perfectly fine on the Window regular terminal, but the split line refactor is OK in this PR.

as per the other PR my assumption is that you are using powershell, but i suggest that we should not document usage for it, because it's a bit non-standard as far as shells go.
FYI, here is how ampersands translate in PS:
https://superuser.com/a/464117